libs/ui-toolkit/src/components/tooltip/tooltip.tsx

```diff
@@ -8,6 +8,7 @@ import {
   Arrow,
   Portal,
 } from '@radix-ui/react-tooltip';
+import type { ITooltipParams } from 'ag-grid-community';
```

Comment:
```suggestion
import type { ITooltipParams as AgTooltipProps } from 'ag-grid-community';
```

Reply:
https://github.com/vegaprotocol/frontend-monorepo/issues/1299 we'll address this here